nucleus/validate/data_transfer_objects/scenario_test_evaluations.py

```diff
@@ -9,3 +11,17 @@ class EvalDetail(ImmutableModel):
 
 class GetEvalHistory(ImmutableModel):
     evaluations: List[EvalDetail]
+
+
+class EvaluationResult(ImmutableModel):
```

Comment:
Let's add the this class to the `validate/__init__.py` imports to allow for easy imports on the client side like `from nucleus.validate import EvaluationResult`

Reply:
good point. Fixed.